ros_cross_compile/docker/build_workspace.sh

```diff
@@ -16,7 +16,8 @@ source /opt/ros/"${ROS_DISTRO}"/setup.bash
 set -ux
 colcon build --mixin "${TARGET_ARCH}"-docker \
   --build-base build_"${TARGET_ARCH}" \
-  --install-base install_"${TARGET_ARCH}"
+  --install-base install_"${TARGET_ARCH}" \
+  --event-handlers console_direct+
```

Comment:
Should this maybe be `console_cohesion+` so that packages don't have interleaved output? The default for ci.ros2.org builds is 

```
--event-handlers console_cohesion+ console_package_list+
```

Reply:
Yeah I guess that works.